lib/parallel_guest_migration_base.pm

```diff
@@ -570,12 +570,21 @@ sub config_host_shared_storage {
             }
             $_temppath = $args{exppath};
             $_temppath =~ s/\//\\\//g;
+            # Create export path directory if it doesn't exist
+            unless (-d $_temppath) {
+                assert_script_run("mkdir -p $_temppath");
+            }
+            # Move guest images from the default image path to the export path
+            # Guest installation puts images to /var/lib/libvirt/images by default,
+            # move them to nfs export directory /home/virt, then mount /home/virt to /var/lib/libvirt/images.
+            assert_script_run("mv $args{mntpath}/* $args{exppath}/");
             assert_script_run("sed -i \'/^.*$_temppath.*\$/d\' /etc/exports");
             assert_script_run("echo \"$args{exppath} *(rw,sync,no_root_squash,no_subtree_check)\" >> /etc/exports");
             assert_script_run("exportfs -a");
             systemctl('restart nfs-server.service');
             systemctl('status nfs-server.service');
             assert_script_run("rm -f -r $args{exppath}/nfsok; touch $args{exppath}/nfsok");
+            assert_script_run("mount -t nfs localhost:$args{exppath} $args{mntpath}");
             save_screenshot;
```

Comment:
In this case, the original image path is mntpath, so there is no wrong to copy images over to exppath from there. But in light of the logic of this change and in order to ensure consistency and general-purpose, I think it would be better to introduce another argument, ```$args{imgpath}```, so
```
 $args{imgpath} //= '/var/lib/libvirt/images';
......
assert_script_run("mv $args{imgpath}/* $args{exppath}/");
```

Reply:
Maybe more general name for example $args{srcpath} is better.

Reply:
Sound good.